Makefile

```diff
@@ -1291,10 +1292,13 @@ ovn_kuttl: input deploy_cleanup ovn ovn_deploy_prep ## runs kuttl tests for the
 
 .PHONY: infra_kuttl_run
 infra_kuttl_run: ## runs kuttl tests for the infra operator, assumes that everything needed for running the test was deployed beforehand.
-	kubectl-kuttl test --config ${INFRA_KUTTL_CONF} ${INFRA_KUTTL_DIR}
+	kubectl-kuttl test --config ${INFRA_KUTTL_CONF} ${INFRA_KUTTL_DIR} --namespace ${INFRA_KUTTL_NAMESPACE}
```

Comment:
$NAMESPACE would be better to keep consistency so I fixed it, though we might want to revisit this because this means infra_kuttl_run and infra_kuttl uses different namespaces by default.